code/modules/client/preference/loadout/loadout_uniform.dm

```diff
@@ -214,113 +214,113 @@
 	path = /obj/item/clothing/under/rank/medical/purple
 	allowed_roles = list("Chief Medical Officer", "Medical Doctor")
 
 /datum/gear/uniform/medical/gscrubs
-	display_name = "medical scrubs, green"
+	display_name = "Medical scrubs, green"
 	path = /obj/item/clothing/under/rank/medical/green
 	allowed_roles = list("Chief Medical Officer", "Medical Doctor")
 
 /datum/gear/uniform/sec
 	subtype_path = /datum/gear/uniform/sec
 
 /datum/gear/uniform/sec/formal
-	display_name = "security uniform, formal"
+	display_name = "Security uniform, formal"
 	path = /obj/item/clothing/under/rank/security/formal
 	allowed_roles = list("Head of Security", "Warden", "Detective", "Security Officer", "Security Pod Pilot")
 
 /datum/gear/uniform/sec/secorporate
-	display_name = "security uniform, corporate"
+	display_name = "Security uniform, corporate"
 	path = /obj/item/clothing/under/rank/security/corp
 	allowed_roles = list("Head of Security", "Warden", "Security Officer", "Security Pod Pilot")
 
 /datum/gear/uniform/sec/dispatch
-	display_name = "security uniform, dispatch"
+	display_name = "Security uniform, dispatch"
 	path = /obj/item/clothing/under/rank/dispatch
 	allowed_roles = list("Head of Security", "Warden", "Security Officer", "Security Pod Pilot")
 
 /datum/gear/uniform/sec/casual
-	display_name = "security uniform, casual"
+	display_name = "Security uniform, casual"
 	path = /obj/item/clothing/under/rank/security2
 	allowed_roles = list("Head of Security", "Warden", "Security Officer", "Detective", "Security Pod Pilot")
 
 /datum/gear/uniform/shorts
 	subtype_path = /datum/gear/uniform/shorts
 
 /datum/gear/uniform/shorts/red
-	display_name = "shorts, red"
+	display_name = "Shorts, red"
 	path = /obj/item/clothing/under/shorts/red
 
 /datum/gear/uniform/shorts/green
-	display_name = "shorts, green"
+	display_name = "Shorts, green"
 	path = /obj/item/clothing/under/shorts/green
 
 /datum/gear/uniform/shorts/blue
-	display_name = "shorts, blue"
+	display_name = "Shorts, blue"
 	path = /obj/item/clothing/under/shorts/blue
 
 /datum/gear/uniform/shorts/black
-	display_name = "shorts, black"
+	display_name = "Shorts, black"
 	path = /obj/item/clothing/under/shorts/black
 
 /datum/gear/uniform/shorts/grey
-	display_name = "shorts, grey"
+	display_name = "Shorts, grey"
 	path = /obj/item/clothing/under/shorts/grey
 
 /datum/gear/uniform/pants
 	subtype_path = /datum/gear/uniform/pants
 
 /datum/gear/uniform/pants/jeans
-	display_name = "jeans, classic"
+	display_name = "Jeans, classic"
 	path = /obj/item/clothing/under/pants/classicjeans
 
 /datum/gear/uniform/pants/mjeans
-	display_name = "jeans, mustang"
+	display_name = "Jeans, mustang"
 	path = /obj/item/clothing/under/pants/mustangjeans
 
 /datum/gear/uniform/pants/bljeans
-	display_name = "jeans, black"
+	display_name = "Jeans, black"
 	path = /obj/item/clothing/under/pants/blackjeans
 
 /datum/gear/uniform/pants/yfjeans
-	display_name = "jeans, Young Folks"
+	display_name = "Jeans, Young Folks"
```

Comment:
```suggestion
	display_name = "Jeans, young folks"
```
what even is this item name? anyway, no caps here maybe?

Reply:
I left that one since the actual item itself is also capitalised. I thought maybe it was a reference I didn't get or something.
```dm
/obj/item/clothing/under/pants/youngfolksjeans
	name = "Young Folks jeans"
	desc = "For those tired of boring old jeans. Relive the passion of your youth!"
	icon_state = "jeansyoungfolks"
	item_color = "jeansyoungfolks"
```